scripts/src/javatests/org/oppia/android/scripts/maven/GenerateMavenDependenciesListTest.kt

```diff
@@ -0,0 +1,423 @@
+package org.oppia.android.scripts.maven
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+import org.mockito.kotlin.doReturn
+import org.mockito.kotlin.eq
+import org.mockito.kotlin.mock
+import org.oppia.android.scripts.common.CommandExecutorImpl
+import org.oppia.android.scripts.proto.License
+import org.oppia.android.scripts.proto.LocalCopyLink
+import org.oppia.android.scripts.proto.MavenDependency
+import org.oppia.android.scripts.proto.MavenDependencyList
+import org.oppia.android.scripts.proto.ScrapableLink
+import org.oppia.android.scripts.testing.TestBazelWorkspace
+import org.oppia.android.testing.assertThrows
+import java.io.ByteArrayOutputStream
+import java.io.File
+import java.io.PrintStream
+
+/** Tests for [GenerateMavenDependenciesList]. */
+class GenerateMavenDependenciesListTest {
+
+  private val DATA_BINDING_POM = "https://maven.google.com/androidx/databinding/" +
+    "databinding-adapters/3.4.2/databinding-adapters-3.4.2.pom"
+  private val PROTO_LITE_POM = "https://repo1.maven.org/maven2/com/google/protobuf/" +
+    "protobuf-lite/3.0.0/protobuf-lite-3.0.0.pom"
+  private val IO_FABRIC_POM = "https://maven.google.com/io/fabric/sdk/android/" +
+    "fabric/1.4.7/fabric-1.4.7.pom"
+  private val GLIDE_ANNOTATIONS_POM = "https://repo1.maven.org/maven2/com/github/" +
+    "bumptech/glide/annotations/4.11.0/annotations-4.11.0.pom"
+  private val FIREBASE_ANALYTICS_POM = "https://maven.google.com/com/google/firebase/" +
+    "firebase-analytics/17.5.0/firebase-analytics-17.5.0.pom"
+
+  private val THIRD_PARTY_PREFIX = "//third_pary:"
+  private val DATA_BINDING_COORD = "androidx.databinding:databinding-adapters:3.4.2"
+  private val PROTO_LITE_COORD = "com.google.protobuf:protobuf-lite:3.0.0"
+  private val GLIDE_ANNOTATIONS_COORD = "com.github.bumptech.glide:annotations:4.11.0"
+  private val FIREBASE_ANALYTICS_COORD = "com.google.firebase:firebase-analytics:17.5.0"
+  private val IO_FABRIC_COORD = "io.fabric.sdk.android:fabric:1.4.7"
+
+  private val DATA_BINDING_COORD_VERSION = "3.4.2"
```

Comment:
I suggest interpolating these versions in the POM links above to keep the two in sync.

Reply:
Done.